app/models/concerns/verification.rb

```diff
@@ -10,32 +10,37 @@ module Verification
     scope :incomplete_verification, -> { where("(users.residence_verified_at IS NULL AND users.failed_census_calls_count > ?) OR (users.residence_verified_at IS NOT NULL AND (users.unconfirmed_phone IS NULL OR users.confirmed_phone IS NULL))", 0) }
   end
 
+  def override(value)
```

Comment:
The way we usually skip the normal behaviour is with a [Guard Clause](http://www.thechrisoshow.com/2009/02/16/using-guard-clauses-in-your-ruby-code/)
How about something like this?:
```
def skip_verification? 
  Setting["feature.user.skip_verification"].present?
end
```
```
def verification_email_sent?
  return true if skip_verification?
  email_verification_token.present?
end
```